index.html

```diff
@@ -97,6 +98,8 @@
         <script src="js/main.js"></script>
 
         <!-- Google Analytics: change UA-XXXXX-X to be your site's ID. -->
+
+          <!-- Fixed The script, removed the original bug -->
```

Comment:
Consider moving google analytics script from `<body>` to `<head>` with adding `<script async src='https://www.google-analytics.com/analytics.js'></script>` for optimizing load times.